mycroft/client/speech/transcribe.py

```diff
@@ -0,0 +1,83 @@
+from mycroft.util.log import LOG
+import wave
+from mycroft.util.signal import (
+    check_for_signal
+)
+import datetime
+import os
+
+
+__author__ = "reginaneon"
+
+text_permission = check_for_signal(
+    'transcribe_text_permission', 0)  # starting default is off
+audio_permission = check_for_signal('keep_audio_permission', 0)
+# text_permission = create_signal('transcribe_text_permission')
+# audio_permission = create_signal('keep_audio_permission')
+
+
+class Transcribe:
+    """
+    Name: Transcribe
+    Purpose: Writes the transcription file.
+    Imports data to the new, more precise text file, containing
+             the lines needed.
+    """
+    @classmethod
+    def write_transcribed_files(self, audio, text):
```

Comment:
Does this have to be in a class? The self reference isn't used.

Reply:
No, it doesn't need to be in a class.  Do you think I should just move write_transcribed_files() over into listener.py?  That works for me...